test/unit/core/simd/simd_debug_stream_test.cpp

```diff
@@ -40,15 +40,14 @@
 #include <iostream>
 #include <sstream>
 
-using namespace seqan3;
 using namespace seqan3::simd;
```

Comment:
no, I meant removing `seqan3::simd`, but keeping `seqan3` :see_no_evil:

Reply:
I think I want to leave it like this, because IF we rip this as an extra module from seqan3. I can easily search and replace :)

Reply:
but if you include seqan3 and not `seqan3::simd` you still don't need to prefix things with `simd::`, because it's an inline namespace. 

I am arguing that the tests should be similar to what people will use in their apps, and people will likely include `seqan3` but I see no-one including `seqan3::simd` and not `seqan3` :thinking:

Reply:
I reverted it